lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -2595,6 +2610,22 @@ def verify_session(session_id, quiet = false)
     end
   end
 
+  #
+  # Custom error code to handle timeout errors
+  #
+  # @param message [String] The message to be printed when a timeout error is hit
+
```

Comment:
Some extra whitespace here:

```suggestion
```

We'd want to add the `@return` type too https://rubydoc.info/gems/yard/file/docs/Tags.md#return

I believe in this case using `Proc` is the best we can do here